datafusion/tests/sql.rs

```diff
@@ -2172,6 +2172,8 @@ async fn csv_explain_analyze() {
     let formatted = arrow::util::pretty::pretty_format_batches(&actual).unwrap();
     let formatted = normalize_for_explain(&formatted);
 
+    println!("ANALYZE EXPLAIN:\n{}", formatted);
```

Comment:
Stray debug logging?

Reply:
Really it is more like a development tool. I will plan to remove it as a follow on PR.